FirebaseTestProject/ViewModels/AuthViewModel.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  AuthViewModel.swift
+//  AuthViewModel
+//
+//  Created by Tim Miller on 8/12/21.
+//
+
+
+import FirebaseAuth
+import SwiftUI
+
+@MainActor class AuthViewModel: ObservableObject {
+  
+  private let auth = Auth.auth()
+  
+  @AppStorage("userSignedIn") var userSignedIn: Bool = false
+  
+  func getUser() -> String {
+    guard let user = auth.currentUser?.email else {
+      return "No User"
```

Comment:
Return `nil` if the `currentUser` is `nil`.  Then the caller can figure out what it wants to display.